src/osha/oira/client/browser/client.py

```diff
@@ -13,12 +13,43 @@ class MailingListsJson(BrowserView):
     """Mailing lists (countries, in the future also sectors and tools)"""
 
     @property
-    def results(self):
+    def paths_to_mailing_lists(self):
+        """Return a list of path/title pairs for given paths, seperated by
+        comma.
+
+        This is used to pre-fill pat-auto-suggest with existing values.
         """
-        List of "mailing list" path/names.
+        q = self.request.get("q", "").strip()
+        if not q:
+            return []
+
+        client_path = "/".join(self.context.getPhysicalPath())
+        paths = list(map(lambda x: f"{client_path}/{x}", q.split(",")))
+        results = []
+        catalog = api.portal.get_tool(name="portal_catalog")
+        if "general" in paths:
```

Comment:
The code is quite obfuscated but it seems to me there is no way to have `"general" in paths` because al the paths have an initial "client_path"